src/Tribe/Linked_Posts/Chooser_Meta_Box.php

```diff
@@ -191,24 +189,23 @@ protected function single_post_dropdown( $linked_post_id ) {
 	 */
 	protected function edit_post_link( $linked_post_id ) {
 		$linked_post_pto = get_post_type_object( $this->post_type );
-		if (
+		$can_edit_others_posts = (
```

Comment:
Perhaps this variable should be: `$cannot_edit_others_posts`?

Reply:
Ahh ... yes. I must have been mixing up _'can'_ and _'cannot'_ - will fix ;-)